buildUtilities/releaseTask.m

```diff
@@ -0,0 +1,63 @@
+function [toolboxOptions] = releaseTask(toolboxVersion)
+%GENERATETOOLBOX Function that generates a toolbox for the boss device API
+
+arguments
+    toolboxVersion (1,:) {mustBeText} = '0.0'
+end
+
+% Toolbox Parameter Configuration
+
+toolboxFolder = "../../bossdevice-api-matlab";
+identifier = "bossdevice-api-matlab";
+toolboxOptions = matlab.addons.toolbox.ToolboxOptions(toolboxFolder, identifier);
+
+toolboxOptions.ToolboxName = "Bossdevice API Toolbox";
+toolboxOptions.ToolboxVersion = toolboxVersion;
+toolboxOptions.Summary = "sync2brain's bossdevice RESEARCH Application Programmable Interface (API) for MATLAB."; 
+toolboxOptions.Description = "For a more detailed description refer to the toolbox README.md file. ↵↵ Contact email: support@sync2brain.com";
+toolboxOptions.AuthorName = "sync2brain";
+toolboxOptions.AuthorEmail = "support@sync2brain.com";
+toolboxOptions.AuthorCompany = "sync2brain";
+toolboxOptions.ToolboxImageFile = "../images/toolboxPackaging.jpg";
+toolboxOptions.ToolboxFiles = ["../images","../toolbox", "../LICENSE", ...
+    "../README.md", "../Bossdeviceapimatlab.prj",...
+     "../buildUtilities", "../resources"];
+currentdir = pwd;
+toolboxOptions.ToolboxMatlabPath = [currentdir(1:end-14),'toolbox'];
```

Comment:
Instead of using pwd, build paths based on the project object. See previous comments.